Update iot-edge-as-gateway.md

kvaes opened the pull request:
Reduce confusion about the opaque gateway reference, when the table references only transparent/protocol/identity

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review and approve here, thanks.

rmca14 commented:
@JennieHubbard

JennieHubbard commented:
#signoff

rmca14 commented:
@sahaj08 
Please resolve the merge conflicts using this guidance https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Resolve%20a%20local%20merge%20conflict%20yourself.aspx, or if this work was done in another pull request, feel free to close this pull request. Thanks.

rmca14 commented:
@sahaj08 
Following up.

sahaj08 commented:
Closing this pull request, I'll re-submit my changes.

kvaes committed:
Update iot-edge-as-gateway.md

Reduce confusion about the opaque gateway reference, when the table references only transparent/protocol/identity
